### PR TITLE
Bugfix/mbe 238 move ip

### PR DESCRIPTION
**Pull request by radireddy**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

## Title of the Pull Requests 
Wrong confirm message shown on confirm page of move IP to any service

### Description of the Change
string "ip_service" is concatenated to a string before translation by mistake. I have removed this and verified.

### Applicable Issues
#MBE-238

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (FredericEspiau) [inline .vscode/settings.json]: You should add `.vscode/*` to the `.gitignore` file

Comment (FredericEspiau) [reply .vscode/settings.json]: (in a difference PR)

File: client/app/ip/ip/block/move/ip-ip-block-move.html [hunk 2]
```diff
@@ -73,12 +73,12 @@
 
             <div data-ng-show="!loading.save">
                 <p class="text-center">
-                    <span data-ng-bind-html="'ip_table_manage_move_ipblock_resume' | translate:{ t0: data.ipBlock.ipBlock, t1: (('ip_service' + model.serviceName.service | translate) || model.serviceName.service) }">
+                    <span data-ng-bind-html="'ip_table_manage_move_ipblock_resume' | translate:{ t0: data.ipBlock.ipBlock, t1: ((model.serviceName.service | translate) || model.serviceName.service) }">
```
Comment (FredericEspiau) [inline client/app/ip/ip/block/move/ip-ip-block-move.html]: `data-translate` and `data-translate-values` would be better :)

**Comment by ovh-cds** [comment]
CDS Report build-manager#768.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✘  
  * Build manager US ✔ 
  * Tests ✔

**Comment by marie-j** [comment]
Could you target `develop` please

File: client/app/ip/ip/block/move/ip-ip-block-move.html [hunk 3]
```diff
@@ -73,13 +73,12 @@
 
             <div data-ng-show="!loading.save">
                 <p class="text-center">
-                    <span data-ng-bind-html="'ip_table_manage_move_ipblock_resume' | translate:{ t0: data.ipBlock.ipBlock, t1: (('ip_service' + model.serviceName.service | translate) || model.serviceName.service) }">
+                    <span data-translate="ip_table_manage_move_ipblock_resume"
+                        data-translate-values="{ t0: data.ipBlock.ipBlock, t1: ((model.serviceName.service | translate) || model.serviceName.service) }">
```
Comment (antleblanc) [inline client/app/ip/ip/block/move/ip-ip-block-move.html]: I'm a bit confused that `t1` is configured to be:

- `model.serviceName.service | translate`
- or `model.serviceName.service`

Does it make sense to translate `model.serviceName.service` value?

File: client/app/ip/ip/block/move/ip-ip-block-move.html (hunk 2 above)
Comment (FredericEspiau) [reply client/app/ip/ip/block/move/ip-ip-block-move.html]: It still is the case :)

**Comment by radireddy** [comment]
@FredericEspiau  and @antleblanc , 

Yes, it does not make sense to translate "model.serviceName.service". I have verified that the translation of this is always null and hence it is showing the value of "model.serviceName.service". I have directly shown the service name on UI. I am not sure why "ip_service" is prepended to the service name before translation. As per the code, this will never work.

**Pull request merged**